test/notebooks/nbtest_transformers.py

```diff
@@ -17,6 +20,7 @@
         'zero_shot_classification.ipynb'
     ]
 )
+@pytest.mark.skip(reason="The expected functionality is not yet implemented in the Transformers Extension")
```

Comment:
I suggest using `xfail` with a reason. This is usually the safest strategy for adding tests for unimplemented things. Additionally, once implemented, the failure of `xfail` will indicate that the test should now be "enabled".

Reply:
With `xfail` the test still runs, which I don't want because it takes a long time.